src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -81,6 +81,18 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Debug,
                 formatString: "Provider {providerType}: Saved stream to {path}");
 
+        private static readonly Action<ILogger, Exception> _noAuthentication =
+            LoggerMessage.Define(
+                eventId: new EventId(13, "NoAuthentication"),
+                logLevel: LogLevel.Warning,
+                formatString: "WARNING: Authentication has been disabled. This can pose a security risk and is not intended for production environments.");
+
+        private static readonly Action<ILogger, Exception> _insecureAuthenticationConfiguration =
+            LoggerMessage.Define(
+                eventId: new EventId(14, "InsecureAutheticationConfiguration"),
+                logLevel: LogLevel.Warning,
```

Comment:
Can we mark this as something higher than Warning?  Warning feels too low for something like this.

Reply:
The only things higher than Warning are Error and Critical. I don't think Critical is appropriate since that is effectively saying the app crashed, but maybe Error is okay. I was preserving the same log level as before this change. @shirhatti  and @wiktork, what do you think?

Reply:
I've added sample output to the PR description.